app/src/main/java/mozilla/lockbox/view/EditItemFragment.kt

```diff
@@ -172,10 +170,7 @@ class EditItemFragment : BackableFragment(), EditItemDetailView {
     }
 
     override fun closeKeyboard() {
-        val imm = context?.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
-        if (imm.isAcceptingText) {
-            imm.hideSoftInputFromWindow(view!!.windowToken, 0)
-        }
+        super.closeKeyboard(view)
```

Comment:
Nit: This method does nothing now; you may remove it.

Reply:
`closeKeyboard()` needs to be called from the Presenter, which is why I kept it here :)